### PR TITLE
session: Ensure bridge has a sane PATH in it's environment

### LINKED ISSUE #1624: $PATH is sometimes "(null)".
This code in cockpit-services.js

```
cockpit.spawn([ "which", "gear" ], { host: me.address })
```

produces this message in the journal

```
which: no gear in ((null))
```

This means that $PATH was lost somewhere.

### PR DESCRIPTION
**Pull request by petervo**
Fixes #1624

### CODE REVIEW

**Comment by stefwalter** [comment]
This looks like it makes sense. But I wonder what happens when ssh is used as the login mechanism instead of cockpit-session? You can check this easily by using ```--local-ssh``` argument on ```cockpit-ws```.

File: src/ws/session.c [hunk 1]
```diff
@@ -903,11 +903,11 @@ main (int argc,
         {
           if (clearenv () != 0)
             err (1, "couldn't clear environment");
-
-          /* set a minimal environment */
-          setenv ("PATH", "/usr/sbin:/usr/bin:/sbin:/bin", 1);
         }
 
+      /* set a minimal environment */
+      setenv ("PATH", "/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin:/sbin:/bin", 1);
```
Comment (mvollmer) [inline src/ws/session.c]: If we add /usr/local (which I think is OK), it should go to the front.

Comment (stefwalter) [inline src/ws/session.c]: Nod.

**Comment by petervo** [comment]
Fixed the order.

When you login with ssh, PATH is determined by the user you are using that machine with. So it will be different. We need to decide if we want to hard code PATH in the bridge to ensure it's always consistent or we want to allow machine specific PATHs.

**Comment by stefwalter** [comment]
@petervo can you file a separate bug about consistent $PATH. I'll merge this, as this change does fix a bug on its own.

**Pull request closed without merging**